src/app/ubs-admin/components/ubs-header/ubs-header.component.html

```diff
@@ -1,13 +1,19 @@
-<div class="wrapper">
+<header>
   <div class="logo">
-    <img src="assets/admin-header/logo-admin.svg" alt="UBS admin home" />
+    <img [src]="ubsHeaderIcons.ubsAdminLogo" alt="UBS admin logo" />
   </div>
-  <div class="icons">
-    <img src="assets/admin-header/first-icon-admin.svg" alt="Selected items" />
-    <img src="assets/admin-header/bell-icon-admin.svg" alt="Notifications" />
-    <span>En</span>
-    <img src="assets/img/arrow_down.svg" alt="Change language" />
-    <a href="/">Home</a>
+  <nav>
+    <a class="nav-link" href="#">Про нас</a>
+    <a class="nav-link" href="#">Правила сортування</a>
+    <a class="nav-link" href="#">Еко-магазин</a>
+    <a class="nav-link" href="#">Green City</a>
```

Comment:
this link should lead to the main page of green city

Reply:
done